services/src/main/java/org/keycloak/protocol/ciba/decoupledauthn/DelegateDecoupledAuthenticationProvider.java

```diff
@@ -179,6 +180,7 @@ public void doBackchannelAuthentication(ClientModel client, BackchannelAuthentic
                 .param(CIBAConstants.SCOPE, request.getScope())
                 .param(DECOUPLED_DEFAULT_CLIENT_SCOPE, defaultClientScope)
                 .param(CIBAConstants.BINDING_MESSAGE, request.getBindingMessage())
+                .param(CIBAConstants.USER_CODE, BooleanUtils.toBoolean(userCodeSupported) ? request.getUserCode() : null)
```

Comment:
Could you realize this line not introducing `BooleanUtils`?

It seems that the following keycloak CI test failed due to introducing this `BooleanUtils`.
https://github.com/tnorimat/keycloak/pull/2/checks?check_run_id=1241420143

Reply:
Removed this library class.

Reply:
I've confirmed. Thank you.